services/web-app/pages/index.js

```diff
@@ -122,8 +121,8 @@ const PageContent = () => {
             <dl>
               <dt className={styles['dashboard-label']}>Components</dt>
               <dd className={styles['dashboard-content']}>
-                Building blocks (such as buttons, links, and dropdown menus) that are pre-built and
-                ready to use when creating new experiences.
+                Building blocks that have been designed and coded to solve a specific user interface
```

Comment:
What about saying "common" instead of "specific"?

Reply:
The intent of that definition (which existed pre-platform) is:

* Component X solves (specific) UI problem A
* Component Y solves (specific) UI problem B

If it where:

* Component X solves (common) UI problem A
* Component Y solves (common) UI problem A

That means we have duplicate components, as both are solving the common UI problem A.

The intent is that each component serves a unique (specific) purpose, and any time that purpose needs addressed, that unique component is reused.

Reply:
I'd like to keep the examples here: "(such as buttons, links, and dropdown menus)". When drafting content for this page, I first didn't have examples and got requests during informal testing. When we did our formal user research, folks often read the examples out loud and then said "oh ok", like it helped clarify what a component is. I'd like to keep the examples there based on our user feedback.

I'd also keep 'specific' -- as in there are correct components to select in a given scenario.

Reply:
@aubrey-oneal in this screenshot, production is on the left and this PR is on the right.

The components description on the production site has three lines, where the other descriptions are all at two lines. Also, the components description includes examples, where the other descriptions do not. Is that the intent?

To me, the components description seemed like an outlier, why I thought it could be cleaned up. I'm happy to proceed with your recommendation; I'm just sharing why I did what I did.

<img width="2560" alt="image" src="https://user-images.githubusercontent.com/1691245/207640687-75e9d97a-ae02-4cfd-8e0b-3987e98d0409.png">

Reply:
If you'd like to shorten to 2 lines perhaps include just 1-2 examples. The formatting is a little different, but I think it does the job well.

Reply:
I reverted so the component description remains outliner in format and length.